tools/README.adoc

```diff
@@ -0,0 +1,121 @@
+[%hardbreaks]
+:toc: left
+:toclevels: 3
+
+
+== Overview
+
+This folder contains the following tools:
+
+. `sgcollect_info` -- a tool that collects diagnostics from Sync Gateway, such as logs, heap profile, cpu profile, expvars, etc.
+. A copy of `cbcollect_info` and `tasks.py`, which was refactored from the upstream `cbollect_info` in the nsserver repository, which has since diverged from this version.  `sgcollect_info` depends on the `cbcollect_info` in this repo.
```

Comment:
(Nitpick) Should be `ns_server` not `nsserver`

Reply:
I linked to the file on the nsserver repo